types/build.rs

```diff
@@ -103,6 +103,15 @@ fn build_anemo_services(out_dir: &Path) {
                 .codec_path("anemo::rpc::codec::BincodeCodec")
```

Comment:
I can't comment on that line, but is the old send_message rpc un-needed after this change?